jest.preset.js

```diff
@@ -4,10 +4,16 @@ const path = require('path');
 
 module.exports = {
   ...nxPreset,
-  testEnvironment: path.join(
-    __dirname,
-    'scripts/gf/test-util',
-    'jsdom-lax-ssl-environment.ts'
-  ),
+  testEnvironment: path.join(__dirname, 'scripts/gf/test-util', 'jsdom-lax-ssl-environment.ts'),
   testURL: 'https://localhost:5001',
+  /* TODO: Update to latest Jest snapshotFormat
+   * By default Nx has kept the older style of Jest Snapshot formats
+   * to prevent breaking of any existing tests with snapshots.
+   * It's recommend you update to the latest format.
+   * You can do this by removing snapshotFormat property
+   * and running tests with --update-snapshot flag.
+   * Example: "nx affected --targets=test --update-snapshot"
+   * More info: https://jestjs.io/docs/upgrading-to-jest29#snapshot-format
+   */
+  snapshotFormat: { escapeString: true, printBasicPrototype: true },
```

Comment:
question: did you try running the example command in the comment above to migrate to the latest format?

Reply:
Yes and it didn't change anything, I guess we aren't using snapshot testing anywhere.